frontend/src/components/Button/index.module.css

```
@@ -15,3 +15,8 @@
   border-radius: 300px;
   box-shadow: 0 4px 3px var(--color-gray);
 }
+
+.button:disabled {
+  color: var(--color-dark-gray);
+  background-color: var(--color-light-gray);
+}
```

Why not use `sass` now that we have it installed?

```suggestion
  box-shadow: 0 4px 3px var(--color-gray);
  
  &:disabled {
    color: var(--color-dark-gray);
    background-color: var(--color-light-gray);
   }
}
```

---

P.S. Not sure if you've tried this already, but if you agree, you can commit my suggestion straight from here. Look for the `Commit suggestion` button

---

That's a good point, I'll do that 🙏